### DISCUSSION TITLE
Full lines in the constellation

### DISCUSSION THREAD

**Discussion by 22kuba22** [Stellarium/stellarium#2258] [Q&A]
![stellarium-001](https://user-images.githubusercontent.com/99351293/153254821-1c9fe4ee-7068-4cbb-8e8d-f6ab74be95b1.jpg)
Hello, I have a question about exporting if the image can be saved other than ctrl-s. And make the constellation lines full?
I haven't figured it out yet. Thank you very much for your help.

**Comment by gzotti** [comment]
What do you mean by "full" lines?

You can configure another key combination to take a screenshot. Other details can be configured in the F2/Tools menu.

**Comment by 22kuba22** [comment]
So that the line in the constellation is one straight line and not composed of several softer lines?
I have already tried in the settings see photo to strengthen the line to 5

**Reply by gzotti** [reply]
Straight lines in a curved projection? Not a good idea.
Line width can be configured in the skycultures dialog.

**Comment by 22kuba22** [comment]
Super thank you very much for your help

**Comment by 22kuba22** [comment]
I would like to use the generated constellation from Stellarium 'for my project, I would like to ask if any license is needed. Or how to secure it, or at least some consent of the author's rights, I am mainly interested in Eastern Europe. Thank you for your help and have a nice day.

**Reply by gzotti** [reply]
What do you mean by "generated constellation"? The lines are coded in terms of star index numbers and stored in text files. Do you mean "generated constellation screenshots"?

About possible license issues: https://github.com/Stellarium/stellarium/wiki/FAQ#can-i-use-stellarium-screenshots-in-my-bookwebsitecalendar-etc

**Comment by 22kuba22** [comment]
![stellarium-000](https://user-images.githubusercontent.com/99351293/153417103-82c79620-1347-476f-9669-47f351e311fd.jpg)
The project is designed to print posters from the star map, see the picture - this would be printed on the poster. I found out that the author of the constellation lines for the culture of the western sky is Mr. Fabien Chereau. And I would like to ask for permission to use. Thank you for your help

**Reply by gzotti** [reply]
See #790

**Comment by 22kuba22** [comment]
Yes # 790 I read it there, but I'd rather have consent directly from him in my question.

**Comment by 22kuba22** [comment]
And won't I need more approval from another author? What do you think see the picture. Otherwise thank you very much for your help !!!

Don't you know how to contact Mr. Fabien Chereau or do you think he will answer me here?

**Reply by gzotti** [reply]
I think all our statements tell you that you can use such images, but that we would appreciate being credited visibly, e.g. a text that says "Created with Stellarium 0.21.3". 
My recommendation is that you disable he Oculars plugin to get rid of the menu in the top-right corner, and press Ctrl-T before taking the screenshot to get rid of the bottom line. 
@xalioth I guess you agree also here?
